Studio-Lovelies/GG-JointJustice-Unity
# Remove test workflow

## Summary

Removes the test run workflow
## Testing instructions

1. Merge this change into `develop` (or fork this repo and merge them into `develop` on your fork)
2. Push code to `develop` or create a PR
3. Notice only builds being executed

## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[x]` Removes existing feature
  - Automated PlayMode and EditMode test runs
- `[ ]` Has associated resource:
